Source/Proximity/GetValueUnfinalized_ProximityPostPatch.cs

```diff
@@ -143,10 +143,6 @@ public static void PostFix(ref float __result, StatWorker __instance, StatDef __
 			{
 				__result += ProximityData.GetProxData(thing, 26);
 			}
-            if (___stat == StatDefOf.UnskilledLaborSpeed)
```

Comment:
Unsure if this should go or not. 
It's not the Offset, but it's the only reference I could find to the UnskilledLabourSpeed base